src/main/views/courts/tabs/openingHoursContent.njk

```diff
@@ -124,12 +153,33 @@
     name: "opening_times[" + opening_times.length +"][hours]"
   }) }}
 
-  {{ govukButton({
-    text: "Clear",
-    name: "clearOpeningTime",
-    type: "button",
-    classes: "govuk-button--secondary clearOpeningTime"
-  }) }}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-three-quarters">
+      {{ govukButton({
+        text: "Clear",
+        name: "clearOpeningTime",
+        type: "button",
+        classes: "govuk-button--secondary clearOpeningTime"
+      }) }}
+    </div>
+    <div class="govuk-grid-column-one-quarter govuk-button-group">
+      {{ govukButton({
+        html: upArrowHtml,
+        name: "moveUp",
+        type: "button",
+        classes: "govuk-button--secondary move-up",
+        attributes: { "aria-label": "move up" }
+      }) }}
+
+      {{ govukButton({
+        html: downArrowHtml,
+        name: "moveDown",
+        type: "button",
+        classes: "govuk-button--secondary move-down",
+        attributes: { "aria-label": "move down" }
+      }) }}
+      </div>
+  </div>
```

Comment:
Re-usable component mentioned in https://github.com/hmcts/fact-admin/pull/241/files#r655551876 could be useful here too